packages/parser/src/components/parser/statement/index.ts

```diff
@@ -31,6 +31,9 @@ export default abstract class Statement {
       case TokenTypes.AGAR_BHAI:
         return BhaiLangModule.getIfStatement();
 
+      case TokenTypes.NAHI_TO_BHAI:
```

Comment:
There is no need to handle this token separately, as this will get handled inside `IfStatement`. Also, there is no need of a separate else if statement altogether. You can take a look how `else` is  handled in consequent part.

Reply:
Yep, this will be removed when I add handling for it in the if statement parser itself

Reply:
This is done